crate/src/page/partial/guide_list.rs

```diff
@@ -172,23 +168,23 @@ fn view_guide_list_item(
     guide: &Guide,
     active: bool,
     matched: bool,
-) -> impl View<Msg> {
+) -> Node<Msg> {
     li![
-        class![
-            C.hover__bg_green_100 => !matched,
-            C.bg_green_200 => matched,
+        C![
+            IF!(!matched => C.hover__bg_green_100),
```

Comment:
Just tip - there is a new `not` helper - e.g. `IF!(not(matched) => ..)`. I use it sometimes to make code more readable. Use as you wish.